src/main/java/org/jenkinsci/test/acceptance/junit/Native.java

```diff
@@ -45,9 +63,34 @@ public void evaluate() throws Throwable {
 
                 private void verifyNativeCommandPresent(Native n) throws IOException, InterruptedException {
                     if (n==null)        return;
+                    
+                    // Checks performed in this annotation only makes sense if Jenkins and Tests are being executed in the same machine
+                    if (!(controller instanceof LocalController)) {
+                        throw new AssumptionViolatedException("Test skipped. Native should be used with a local controller, otherwise it cannot be fully trusted to work as expected.");
```

Comment:
So I was going to do that - but in order to fix it properly you need to have Jenkins running - the point of Native was to avoid the overhead of starting Jenkins.

However I am fine if we decide that reliably detecting tools on the SUT is better than waving our arms in the air and doing nothing more than guessing.